app/views/search-for-your-gp.nunjucks

```diff
@@ -16,7 +24,7 @@
 </ul>
 <details class="" role="group">
   <summary data-analytics="summary" role="button" aria-controls="details-content-0" aria-expanded="false">
-    <span class="details__summary">If you haven't registered for online services</span>
+    <span class="details__summary" onmousedown="dcsMultiTrack('GPAppointment', 'Not_Registered', 'WT.dl', '121')">If you haven't registered for online services</span>
```

Comment:
All of the analytics stuff is inline (view-source:https://beta.nhs.uk/finders/find-help). There is a rational for that in usage instructions for GA, WebTrens, etc.

I'd suggest we accept this PR and have a separate discussion about the pros and cons of changing this approach.

@beseven - are you OK with this?